hybris/custom/klarnapayment/src/com/klarna/payment/converter/populator/KPCreditSessionAttPopulator.java

```diff
@@ -135,15 +135,16 @@ public void populate(final AbstractOrderModel source, final PaymentsSession targ
 		LogHelper.debugLog(LOG, "inside full populator");
```

Comment:
No debug log on production code

Reply:
Debug logs have been removed
Commit can be found in https://github.com/klarna/SAP-Commerce-klarna-payments/commit/c3d029e9830e600544e03277455cf385023f4a5c